src/Datasource/EntityTrait.php

```diff
@@ -1207,7 +1207,7 @@ public function getInvalid()
      * Get a single value of an invalid field. Returns null if not set.
      *
      * @param string $field The name of the field.
-     * @return mixed
+     * @return mixed|null
```

Comment:
woops

Reply:
having null as verbose type here actually makes nullable part more clear, dont u think?

Reply:
Meh, if the static analyzers stay happy I don't care.